tests/beaker/pre-suite/02_add_extra_hdd.rb

```diff
@@ -0,0 +1,34 @@
+test_name 'Add extra hard drive for LVM testing'
+
+
+# Get the auth_token from ENV
+auth_tok = ENV['AUTH_TOKEN']
```

Comment:
You'll want to force a failure if the token is not set otherwise a test run failure because of a missing token is hard to discover.